app/assets/stylesheets/static_pages.scss

```diff
@@ -153,6 +153,10 @@ label.required:after {
   }
 }
 
+input[type=checkbox][disabled]{
+   width: initial;
```

Comment:
This only fixes the issue when the box is disabled. We also need it to be fixed when the box is enabled.

I think removing the [disabled] fixes it everywhere.